SCClassLibrary/Common/Control/Server.sc

```diff
@@ -153,8 +161,8 @@ ServerOptions {
 				o = o ++ " -T " ++ threads;
 			}
 		});
-		if (useSystemClock.notNil, {
-			o = o ++ " -C " ++ useSystemClock.asInteger
+		if (useSystemClock, {
```

Comment:
if you want to be sure this is backward compatible, you could write `useSystemClock.asBoolean`

Reply:
`useSystemClock` can be set to `true` or `false` (default) according to the documentation. I'm not certain that allowing `nil` to act as `false` was such a good idea in the first place. In addition, calling `asBoolean` would allow numbers to be treated as `true/false`. I'd rather we limit the number of acceptable values to literal booleans.

Reply:
@brianlheim is this also a breaking change?

Reply:
i think if the documentation is clear about the expected type, and there's no existing pattern we know of of abusing it, it's fair game